packages/forms/examples/forms/pages/datepicker/datepicker.page.ts

```diff
@@ -1,7 +1,70 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+import { DateRange } from '@acpaas-ui/js-date-utils';
 
 @Component({
-	templateUrl: './datepicker.page.html',
+  templateUrl: './datepicker.page.html',
```

Comment:
Fix indenting please. On a sidenote, I don't get why your code editor is doing this by default, maybe you should check your settings.